syn/yosys-plugins/fasm_build.patch

```diff
@@ -0,0 +1,69 @@
+diff --git a/fasm-plugin/Makefile b/fasm-plugin/Makefile
```

Comment:
Can you just send this patch upstream?

Reply:
Unfortunately it is Windows specific - mingw in conda compiles to `.o` file instead of `.so`.
Other change that is there, is converting paths to Unix format (basically changing '\' to '/'). This one won't harm other os compilation, but without full windows support it does not make sense to add it

Reply:
The upstream plugin should support windows.

Reply:
This patch would cause Linux/Osx build failure. I will prepare portable Makefile or separate Makefile for Windows - whatever turns out to be cleaner solution.

Reply:
This patch will cause build failure on Linux/Osx. I will prepare portable Makefile or separate Makefile for Windows - whatever turns out to be cleaner solution.